diff --git a/node_modules/grunt-webfont/tasks/engines/fontforge/generate.py b/node_modules/grunt-webfont/tasks/engines/fontforge/generate.py
index ffefdf0..6309ba6 100644
--- a/node_modules/grunt-webfont/tasks/engines/fontforge/generate.py
+++ b/node_modules/grunt-webfont/tasks/engines/fontforge/generate.py
@@ -124,7 +124,7 @@ if 'woff' in args['types']:
 # EOT
 if 'eot' in args['types']:
 	# eotlitetool.py script to generate IE7-compatible .eot fonts
-	call('python "%(path)s/../../bin/eotlitetool.py" "%(font)s.ttf" --output "%(font)s.eot"' % {'path': scriptPath, 'font': fontfile}, shell=True)
+	call('python2 "%(path)s/../../bin/eotlitetool.py" "%(font)s.ttf" --output "%(font)s.eot"' % {'path': scriptPath, 'font': fontfile}, shell=True)
 
 # Delete TTF if not needed
 if (not 'ttf' in args['types']) and (not 'woff2' in args['types']):
